beacon_node/beacon_processor/src/lib.rs

```diff
@@ -1142,6 +1131,10 @@ impl<E: EthSpec> BeaconProcessor<E> {
                             self.spawn_worker(item, idle_tx);
                         } else if let Some(item) = blbroots_queue.pop() {
                             self.spawn_worker(item, idle_tx);
+                        } else if let Some(item) = dcbroots_queue.pop() {
```

Comment:
Looks like the intent before was to prioritize all by roots requests ahead of all by range requests? 

> // Prioritize by_root requests over by_range as the former are time sensitive for recovery

Reply:
I have no evidence this is useful, so let's just extend the deneb priorization